packages/flutter/test/material/filter_chip_test.dart

```diff
@@ -840,7 +840,7 @@ void main() {
     // Test default icon theme.
     await tester.pumpWidget(buildChip());
 
-    expect(getIconData(tester).color, ThemeData().iconTheme.color);
+    expect(getIconData(tester).color, theme.colorScheme.primary);
```

Comment:
This actually checks the theme value so the theme variable is good here.

Reply:
Do you mean in comparison with the other comment? To use or not the `final theme = ThemeData()` variable?

Reply:
If the test checks `theme.{value}`, keep the variable.